SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -273,6 +273,7 @@ void CIncEulerSolver::SetNondimensionalization(CConfig *config, unsigned short i
 
   config->SetTemperature_Ref(1.0);
   config->SetViscosity_Ref(1.0);
+  config->SetConductivity_Ref(1.0);
```

Comment:
This was not needed at the moment, if you want to add it please do so also for the other solvers.

To clarify on my comment, do composition-dependent properties work with non-dimensional simulations? or not?
If they don't and the fix is not easy we need to throw an error.
If they do we need a regression test.

Reply:
Thank you so much for the explanation. I will check whether it works for Non-dimensional simulations.